src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -14,17 +16,40 @@ const initLoginForm = () => {
 
     overrideSwaggerAuthorizeEvent(swagger);
     overrideSwaggerLogoutEvent(swagger);
+    loadLastSession(swagger);
     showLoginUI(swagger);
 }
 
+const loadLastSession = (swagger) => {
+    if (isAuthorized(swagger))
+        return;
+
+    const lastToken = window.localStorage.getItem(TOKEN_SWAGGER_NAME);
+    if (!lastToken)
+        return;
+
+    const obj = generateAuthorizeObjectForSwagger(lastToken);
```

Comment:
please choose a better name for this variable or completely remove it (use its value directly).